powerdns/utils.py

```diff
@@ -17,6 +19,43 @@ class Meta:
         abstract = True
 
 
+class Owned(models.Model):
+    """Model that has an owner. This owner is set as default to the creator
+    of this model, but can be overridden."""
+
+    owner = models.ForeignKey('auth.User', null=True, blank=True)
+
+    class Meta:
+        abstract = True
+
+    def email_owner(self, creator):
```

Comment:
Do we really need to cover this? Especially that we plan to use it as microservice more and less as a django app.